examples/tutorials/analysis-1d/cta_sensitivity.py

```diff
@@ -49,6 +49,7 @@
 # Check setup
 # -----------
 from gammapy.utils.check import check_tutorials_setup
+from cycler import cycler
```

Comment:
I think cycler import should be moved above with the rest of the imports. Also in other tutorials, the check tutorials setup import has been moved with the rest of the imports.